docs/api/scheduler.md

```diff
@@ -646,6 +646,31 @@ In the example below there are three allocations belonging to two applications,
 
 **Code** : `500 Internal Server Error`
 
+### Queue applications by state
+
+Fetch all Applications for the given Partition/Queue/State combination and displays general information about the applications like used resources, queue name, submission time and allocations. Only following application states are allowed: active, rejected, completed. Active is a fake state that represents all application states except completed and rejected. For active state, can narrow the result by status query parameters(case-insensitive). For example, can fetch `Running` applications for the default partition and root queue by `/ws/v1/partition/default/queue/root/applications/active?status=running`. In case the queue name contains any special characters, it needs to be url escaped to avoid issues.
```

Comment:
> Only following application states are allowed: active, rejected, completed.

`rejected` and `completed` are not acceptable, right?

Reply:
Sorry about the mistake. The error has been corrected, and I've made some changes for more clarity regarding the active state.